paymentsheet/src/main/java/com/stripe/android/customersheet/ui/CustomerSheetScreen.kt

```diff
@@ -84,23 +87,42 @@ internal fun SelectPaymentMethod(
 
 @Composable
 internal fun SelectPaymentMethodContent(
-    header: String?,
+    viewState: CustomerSheetViewState.SelectPaymentMethod,
+    viewActionHandler: (CustomerSheetViewAction) -> Unit,
 ) {
     val horizontalPadding = dimensionResource(R.dimen.stripe_paymentsheet_outer_spacing_horizontal)
 
     Column {
         H4Text(
-            text = header ?: stringResource(
+            text = viewState.title ?: stringResource(
                 R.string.stripe_paymentsheet_select_payment_method
             ),
             modifier = Modifier
                 .padding(bottom = 2.dp)
                 .padding(horizontal = horizontalPadding)
         )
 
-        Text(
-            text = "Hello world!",
-            modifier = Modifier.padding(horizontal = horizontalPadding)
+        PaymentOptions(
+            state = PaymentOptionsState(
+                items = viewState.paymentMethods,
+                selectedIndex = viewState.paymentMethods.indexOfFirst {
+                    it.paymentMethod.id == viewState.selectedPaymentMethodId
+                },
+            ),
+            isEditing = viewState.isEditing,
+            isProcessing = viewState.isProcessing,
+            onAddCardPressed = { viewActionHandler(CustomerSheetViewAction.OnAddCardPressed) },
```

Comment:
What are you thinking for the ideal future state here? If PS/FC used the `viewActionHandler`, would they just emit events to it, or would we still have separate handlers for each of these actions?

Reply:
Yea, I am thinking that each screen would have their own `viewActionHandlers` that take in their own `*ViewAction` as an argument. Do you have any other suggestions?

Reply:
Nope that's the direction I was thinking too! Certainly open to other things as we explore and see what works best though!